components/auth/SignUpView.tsx

```diff
@@ -34,9 +34,11 @@ const SignUpView: FC<Props> = () => {
       setMessage('')
       await signup({
         email,
-        firstName,
-        lastName,
         password,
+        metadata: [
```

Comment:
Unfortunately, we cannot make this changes in this branch. They should be added to the upstream branch of `next-commer:master`